source/services/ManifestFactory.js

```diff
@@ -535,6 +535,11 @@ class ManifestFactory extends libFableServiceProviderBase
 			tmpSection.Name = tmpSectionName;
 		}
 
+		if (tmpSection['Section CSS'])
+		{
+			tmpSection.CSSClass = tmpSection['Section CSS'];
```

Comment:
(nit) May be fine if this matches the existing convention, but IMO this CSV heading reads like it will include CSS rules, not a class. Is there a case where people using the CSV will want to specify this? Is this just for maintaining parity with CSV and JSON?